python/mxnet/foo/nn/layer.py

```diff
@@ -207,6 +222,80 @@ def generic_forward(self, F, x, *args, **kwargs):
         raise NotImplementedError
 
 
+class _Binary(Layer):
```

Comment:
I agree. I have removed operator overloading. Only the explicit form (`Add(lhs, rhs)`) remains now.